### PR TITLE
HAP-1207 - Support multiple paths in Artifactory trigger

### PR DESCRIPTION
**Pull request by AlexeiVainshtein**

### CODE REVIEW

File: src/main/webapp/help/Trigger/help-pathsToWatch.html [hunk 1]
```diff
@@ -16,4 +16,5 @@
 `
```
Comment (Kuanlin-Chen) [inline src/main/webapp/help/Trigger/help-pathsToWatch.html]: Would you mind deleting this symbol ?

File: src/main/webapp/help/Trigger/help-pathsToWatch.html [hunk 2]
```diff
@@ -16,4 +16,5 @@
...
 <div>
```
Comment (eyalbe4) [inline src/main/webapp/help/Trigger/help-pathsToWatch.html]: As @Kuanlin-Chen suggested, let'e remove the dot above this line.

Comment (AlexeiVainshtein) [reply src/main/webapp/help/Trigger/help-pathsToWatch.html]: Done

File: src/main/webapp/help/Trigger/help-pathsToWatch.html (hunk 1 above)
Comment (AlexeiVainshtein) [reply src/main/webapp/help/Trigger/help-pathsToWatch.html]: Done

**Pull request merged**